File: questionTwo java/Circle.java
import java.util.Scanner;

public class Circle {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.print("Enter the radius of the circle: ");
        int radius = input.nextInt();

        System.out.printf("Diameter: %d%n", 2 * radius);
        System.out.printf("Circumference: %.2f%n", 2 * Math.PI * radius);
        System.out.printf("Area: %.2f%n", Math.PI * radius * radius);
    }
}

File: Chapter 2/circle.c
#include <stdio.h>

int circleProperties(float radius, float *circumference, float *area) {
    const float PI = 3.14159;
    *circumference = 2 * PI * radius;
    *area = PI * radius * radius;
    return 0;
}

int main() {
    float radius;
    float circumference, area;

    printf("Enter the radius of the circle: ");
    scanf("%f", &radius);

    circleProperties(radius, &circumference, &area);

    printf("Circumference: %.2f\n", circumference);
    printf("Area: %.2f\n", area);

    return 0;
}